### DISCUSSION TITLE
TUTORÍA PRG1 JUEVES 17/10 10.30h.

### DISCUSSION THREAD

**Discussion by Daniel-Cotarelo** [mmasias/24-25-PRG1#452] [Polls]
TUTORÍA PRG1 JUEVES 17/10 10.30h.

**Comment by mmasias** [comment]
Estupendo... 

Por favor, además de votar, indicar los nombres de los que se anotan para la tutoría (lo que no quita que si alguien quiere apuntarse a última hora lo haga). Asimismo, recordar que la naturaleza de las tutorías es aclarar las dudas que puedan ir surgiendo del desarrollo de las clases, por lo que conviene que vayan con dudas y con el material de las mismas revisado, repasado y habiendo hecho un razonable esfuerzo por entenderlo. (O sea: **repasar esta noche!!!**)

Un saludo!

Manuel
